### PR TITLE
style the Q&A page

### PR DESCRIPTION
**Pull request by SamChun**
created new template qanda: template-qanda.php
style page

### CODE REVIEW

File: wordpress/wp-content/themes/pfennings/src/sass/layout/_content.scss [hunk 1]
```diff
@@ -413,3 +413,31 @@ section[role=main] article.pricelist {
...
+	border-left: none;
+	width: 100%;
+	padding: 0 2em;
+
+	@include media($bp-large) {
+		@include span-columns(8);
+		@include shift(2);
+	}
+
+	h4 {
```
Comment (lmann) [inline wordpress/wp-content/themes/pfennings/src/sass/layout/_content.scss]: Hi Sam

All typography elements should be in the typography partial

File: wordpress/wp-content/themes/pfennings/src/sass/layout/_content.scss [hunk 2]
```diff
@@ -413,3 +413,31 @@ section[role=main] article.pricelist {
...
+		margin-top: 2em;
+		margin-bottom: 0.75em;
+	}
+
+	h3 {
```
Comment (lmann) [inline wordpress/wp-content/themes/pfennings/src/sass/layout/_content.scss]: same here

File: wordpress/wp-content/themes/pfennings/src/sass/layout/_content.scss [hunk 3]
```diff
@@ -413,3 +413,31 @@ section[role=main] article.pricelist {
...
+		margin-top: 2em;
+	}
+
+	ul {
```
Comment (lmann) [inline wordpress/wp-content/themes/pfennings/src/sass/layout/_content.scss]: and here

File: wordpress/wp-content/themes/pfennings/template-qanda.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<?php
```
Comment (lmann) [inline wordpress/wp-content/themes/pfennings/template-qanda.php]: this looks very similar to the base page template. Do w really need a new template for this?

Comment (SamChun) [inline wordpress/wp-content/themes/pfennings/template-qanda.php]: The content was wrapped in an article tag. The article tag has styles that are used on other pages. I needed a way to target it to remove the width and border-left without effecting other pages.

File: wordpress/wp-content/themes/pfennings/src/sass/layout/_content.scss (hunk 1 above)
Comment (SamChun) [inline wordpress/wp-content/themes/pfennings/src/sass/layout/_content.scss]: Will move to the typography partial.

**Pull request merged**